### PR TITLE
Update: Calendar nav

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
calendar couldn't change center

## Proposed Changes
ember set

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/addon/components/navi-date-picker.js [hunk 1]
```diff
@@ -166,7 +166,8 @@ class NaviDatePicker extends Component {
    */
   @action
   setCenterDate({ moment: newDate }) {
-    this.centerDate = newDate;
+    debugger;
```
Comment (daniellevinson) [inline packages/reports/addon/components/navi-date-picker.js]: remove ^

Comment (kevinhinterlong) [reply packages/reports/addon/components/navi-date-picker.js]: Grr always forget to save with vscode

File: packages/reports/tests/integration/components/navi-date-picker-test.js [hunk 2]
```diff
@@ -89,6 +89,26 @@ module('Integration | Component | Navi Date Picker', function(hooks) {
...
+    assert.equal(getMonth(), 'July', 'The center date starts as the passed in date');
+
+    await click('.ember-power-calendar-nav-control--previous');
+    assert.equal(getMonth(), 'June', 'The center date changes to previous month');
+
+    await click('.ember-power-calendar-nav-control--next');
+    assert.equal(getMonth(), 'July', 'The center date goes back to original after click');
+
+    await click('.ember-power-calendar-nav-control--next');
+    assert.equal(getMonth(), 'August', 'The center date goes one month ahead');
```
Comment (mattgraham1995) [inline packages/reports/tests/integration/components/navi-date-picker-test.js]: Do we test changing the month through clicking the month name and selecting a month from the list?

Comment (kevinhinterlong) [reply packages/reports/tests/integration/components/navi-date-picker-test.js]: Nope

Comment (kevinhinterlong) [reply packages/reports/tests/integration/components/navi-date-picker-test.js]: @mattgraham1995 https://stackoverflow.com/q/430237? Doesn't look like it's easily done unless we switch to basic dropdown/etc

Comment (mattgraham1995) [reply packages/reports/tests/integration/components/navi-date-picker-test.js]: It wouldn't even work in an acceptance test?

Comment (kevinhinterlong) [reply packages/reports/tests/integration/components/navi-date-picker-test.js]: Nope, can't trigger it to open, but I was able to do manually change the selected item and then trigger a `'change'` event

```js
const monthSelect = getMonthSelect();
monthSelect.selectedIndex = monthSelect.selectedIndex - 1;
await triggerEvent(monthSelect, 'change');
```

File: packages/reports/tests/integration/components/navi-date-picker-test.js [hunk 3]
```diff
@@ -109,6 +108,32 @@ module('Integration | Component | Navi Date Picker', function(hooks) {
...
+    assert.equal(getMonth(), 'July', 'The center date starts as the passed in date');
+
+    const monthSelect = getMonthSelect();
+    monthSelect.selectedIndex = monthSelect.selectedIndex - 1;
+    await triggerEvent(monthSelect, 'change');
+
+    assert.equal(getMonth(), 'June', 'The center date changes to previous month');
+
+    // change year
+    assert.equal(getYear(), '2015', 'The center date changes to next year');
```
Comment (mattgraham1995) [inline packages/reports/tests/integration/components/navi-date-picker-test.js]: Assert message needs updating

Comment (kevinhinterlong) [reply packages/reports/tests/integration/components/navi-date-picker-test.js]:
```suggestion
    assert.equal(getYear(), '2015', 'The center date year starts as the passed in date');
```

**Pull request merged**